apps/dashboard/src/app/widgets/edit/edit-widget.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'variables';
+@import 'apps/dashboard/src/variables';
```

Comment:
just to know, is it best to use absolute path ?

Reply:
oh good catch, I didn't notice that change which occurred automatically when I moved the files in the folder. I'll revert that as we want to keep `@import 'variables';`.